cmd/operator-sdk/bundle/validate.go

```diff
@@ -132,33 +171,72 @@ func (c bundleValidateCmd) run() (err error) {
 		fields["container-tool"] = c.imageBuilder
 	}
 	logger := log.WithFields(fields)
-	if viper.GetBool(flags.VerboseOpt) {
-		log.SetLevel(log.DebugLevel)
+	if c.json {
+		log.SetLevel(log.PanicLevel)
```

Comment:
I think do something clever with [`JSONFormatter`](https://godoc.org/github.com/sirupsen/logrus#JSONFormatter).

Reply:
It is not a possible option. See: https://github.com/sirupsen/logrus/blob/master/logrus.go#L93-L112

And then, to change this implementation we would need to change the operator-framework/operator-registry. See : https://github.com/operator-framework/operator-registry/blob/master/pkg/lib/bundle/interfaces.go#L24

So, shows out of scope.

Reply:
We're directly setting the `*log.Entry`, so we can modify the [underlying logger](https://github.com/sirupsen/logrus/blob/master/entry.go#L52).

Reply:
Oh. Really good, sorry. I did not know that . Done. Very good spot.